services/auth/test/clients.json

```diff
@@ -16709,7 +16709,7 @@
   },
   {
     "clientId": "mozilla-auth0/ad|Mozilla-LDAP|pmoore/fetch-worker-type-definitions",
-    "description": "This client is used by pmoore's local cron job that fetches worker type definitions, and keeps a git repository of their history.\n\nSee [script](https://github.com/petemoore/myscrapbook/blob/master/sync-worker-type-definitions.sh) and [code](https://github.com/taskcluster/generic-worker/blob/master/worker_types/all-worker-types/main.go).",
+    "description": "This client is used by pmoore's local cron job that fetches worker type definitions, and keeps a git repository of their history.\n\nSee [script](https://github.com/petemoore/myscrapbook/blob/master/sync-worker-type-definitions.sh) and [code](https://github.com/taskcluster/taskcluster/blob/master/worker_types/all-worker-types/main.go).",
```

Comment:
@petemoore non of those links are longer valid. Probably safe to trim this description to exclude urls at all.

Reply:
It looks like that clients.json is some kind of production dump at some point. I wonder if we can just remove that client entirely and other things that are dated.